pkgs/development/tools/database/prisma-engines/default.nix

```diff
@@ -0,0 +1,50 @@
+{ fetchFromGitHub
+, lib
+, openssl
+, pkg-config
+, protobuf
+, rustPlatform
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "prisma-engines";
+  version = "2.30.0";
+
+  src = fetchFromGitHub {
+    owner = "prisma";
+    repo = "prisma-engines";
+    rev = version;
+    sha256 = "sha256-i4r+TRC8454awbqe35Kg3M9xN2NnP8Sbd/dITtm9MDg=";
+  };
+
+  cargoPatches = [
+    # Remove test from compilation targets:
+    # they add time to an already long compilation and some fail out-of-the-box.
+    ./no_tests.patch
+  ];
+
+  # Use system openssl.
+  OPENSSL_NO_VENDOR = 1;
+
+  cargoSha256 = "sha256-FiXtgjS/A8bjUJT5eJG7KSmWIMabvuasR109wvlHRNQ=";
+
+  nativeBuildInputs = [ pkg-config ];
+  buildInputs = [ openssl protobuf ];
+
+  preBuild = ''
+    export PROTOC=${protobuf}/bin/protoc
+    export PROTOC_INCLUDE="${protobuf}/include";
+    export OPENSSL_DIR=${lib.getDev openssl}
+    export OPENSSL_LIB_DIR=${openssl.out}/lib
+  '';
```

Comment:
Add a few more, from our own build setup:

```bash
export SQLITE_MAX_VARIABLE_NUMBER=250000
SQLITE_MAX_EXPR_DEPTH=10000
```

So SQLite works as it does for the other targets.